public/apps/configuration/panels/service-account-list.tsx

```diff
@@ -0,0 +1,218 @@
+      render: truncatedListView(tableItemsUIProps),
+    },
+    {
+      field: 'attributes',
+      name: 'Attributes',
+      render: dictView,
+      truncateText: true,
+    },
+  ];
+}
+
+export function ServiceAccountList(props: AppDependencies) {
+  const [userData, setUserData] = React.useState<InternalUsersListing[]>([]);
+  const [errorFlag, setErrorFlag] = React.useState(false);
+  const [selection, setSelection] = React.useState<InternalUsersListing[]>([]);
+  const [currentUsername, setCurrentUsername] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [query, setQuery] = useState<Query | null>(null);
+
+  React.useEffect(() => {
+    const fetchData = async () => {
+      try {
+        setLoading(true);
+        const userDataPromise = getUserList(props.coreStart.http, ResourceType.serviceAccounts);
+        setCurrentUsername((await getAuthInfo(props.coreStart.http)).user_name);
+        setUserData(await userDataPromise);
+      } catch (e) {
+        console.log(e);
+        setErrorFlag(true);
+      } finally {
+        setLoading(false);
+      }
+    };
+
+    fetchData();
+  }, [props.coreStart.http]);
+
+  const actionsMenuItems = [
+    <EuiButtonEmpty
+      data-test-subj="edit"
+      key="edit"
+      onClick={() => {
+        window.location.href = buildHashUrl(ResourceType.users, Action.edit, selection[0].username);
+      }}
+      disabled={selection.length !== 1}
+    >
+      Edit
+    </EuiButtonEmpty>,
+    <EuiButtonEmpty
+      data-test-subj="duplicate"
+      key="duplicate"
+      onClick={() => {
+        window.location.href = buildHashUrl(
+          ResourceType.users,
+          Action.duplicate,
+          selection[0].username
+        );
+      }}
+      disabled={selection.length !== 1}
+    >
+      Duplicate
+    </EuiButtonEmpty>,
+    <EuiButtonEmpty
+      key="export"
+      disabled={selection.length !== 1}
+      href={
+        selection.length === 1
+          ? `${props.coreStart.http.basePath.serverBasePath}${API_ENDPOINT_SERVICEACCOUNTS}/${selection[0].username}`
+          : ''
+      }
+      target="_blank"
+    >
+      Export JSON
+    </EuiButtonEmpty>,
+  ];
+
+  const [actionsMenu, closeActionsMenu] = useContextMenuState('Actions', {}, actionsMenuItems);
+
+  return (
+    <>
+      <EuiPageHeader>
+        <EuiTitle size="l">
+          <h1>Service accounts</h1>
+        </EuiTitle>
+      </EuiPageHeader>
+      <EuiPageContent>
+        <EuiPageContentHeader>
+          <EuiPageContentHeaderSection>
+            <EuiTitle size="s">
+              <h3>
+                Service accounts
+                <span className="panel-header-count">
+                  {' '}
+                  ({Query.execute(query || '', userData).length})
+                </span>
+              </h3>
+            </EuiTitle>
+            <EuiText size="s" color="subdued">
+              Here you list the users authenticated via an external authentication system such as
```

Comment:
Is this comment applicable to service accounts?

Reply:
@cwperks  "Here you list the users authenticated via an external authentication system"

I thought that this was the definition of a service account.   

Do you know where can i find  the definition?

Reply:
@samuelcostae You can find a description here (https://gist.github.com/cwperks/e756e1cead72cd511d819241a11337e8#service-account--service-account-tokens). Service account tokens are issued and authenticated by the security plugin so there are no external authentication systems at play here.